README.md

```diff
@@ -1,37 +1,76 @@
 
-* Better folder structure
-* Dependency management with [Composer](http://getcomposer.org)
-* Easy WordPress configuration with environment specific files
-* Environment variables with [Dotenv](https://github.com/vlucas/phpdotenv)
-* Autoloader for mu-plugins (use regular plugins as mu-plugins)
-* Enhanced security (separated web root and secure passwords with [wp-password-bcrypt](https://github.com/roots/wp-password-bcrypt))
+## Requirements
 
-Use [Trellis](https://github.com/roots/trellis) for additional features:
+* PHP >= 5.5
+* MySQL
+* Composer - [Install](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx)
+* Node.js - [Install](https://nodejs.org/en/)
 
-* Easy development environments with [Vagrant](http://www.vagrantup.com/)
-* Easy server provisioning with [Ansible](http://www.ansible.com/) (Ubuntu 14.04, PHP 5.6 or HHVM, MariaDB)
-* One-command deploys
+## Getting Started
 
-See a complete working example in the [roots-example-project.com repo](https://github.com/roots/roots-example-project.com).
+### 0. Set up your environment
 
-## Requirements
+#### Composer
 
-* PHP >= 5.5
-* Composer - [Install](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx)
+Composer is a dependency manager for PHP.
+
+[Install](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx)
+
+#### Node.js + npm
+
+We use Node.js and npm to build the frontend.
+
+[Install](https://nodejs.org/en/)
+
+
+#### Web Server
+> Skip this if you already have a web server set up.
+
+Choose a development environment setup that works best for you.
+
+Some options:
+
+* [Trellis](https://roots.io/trellis/docs/installing-trellis/)
+* [MAMP](https://www.mamp.info)
+* [XAMPP](https://www.apachefriends.org/index.html)
+* [Homestead](https://laravel.com/docs/5.6/homestead)
+
+### 1. Clone the git repo
+```
+git clone https://github.com/CNHCircleK/cnhcirclek.org.git
+```
+
+### 2. Install Composer dependencies
 
-## Installation
+Change to the directory that was created from the clone.
 
-1. Clone the git repo - `git clone https://github.com/roots/bedrock.git`
-2. Run `composer install`
-3. Copy `.env.example` to `.env` and update environment variables:
+```
+composer install
+```
+
+Running this will install the dependencies that the website requires.
+
+[**What is Composer?**](https://getcomposer.org/doc/00-intro.md)
+
+### 3. Define environment variables
+
+> Environment variables are values specific to the current running instance of the project. Having environment variables allows us adapt to different development environments and prevents us from committing secrets such as the database password.
+
+```
+cp .env.example .env
+```
+
+`.env.example` provides a template for your `.env`. The website will refer to `.env`. Notice that we do not commit the `.env` file and it its included in the `.gitignore`.
+
+**Environment Variables**
```

Comment:
How do we determine what values to assign the environment variables?

Reply:
They're defined based on however you set up your MySQL server, etc. There's descriptions for each of the keys.